CONTRIBUTING.md

```diff
@@ -0,0 +1,724 @@
+
+1. You have [submitted a new package](#submitting-a-package-to-biojulia) to BioJulia.
+2. [Reported a bug](#reporting-bugs).
+3. [Suggested enhancements](#suggesting-enhancements).
+4. [Made one or more pull requests](#pull-requests) implementing one or more...
+    - Fixed bugs.
+    - Improved performance.
+    - Added new functionality.
+    - Increased test coverage.
+    - Improved documentation.
+
+None of these requirements are set in stone, but we prefer you to have done one
+or more of the above, as it gives good confidence that you are familiar with the
+tasks and responsibilities of maintaining a package used by others, and are
+willing to do so.
+Any other avenue for demonstrating commitment to the community and the
+GitHub organisation will also be considered.
+
+### BioJulia members can sometimes become administrators
+
+Members of the admin team have often been contributing to BioJulia for a long
+time, and may even be founders present at the inception of the project.
+In order to become an admin, one does not necessarily have to contribute large
+amounts of code to the project.
+Rather the decision to on-board a member to an admin position requires a history
+of using and contributing to BioJulia, and a positive
+interaction and involvement with the community. Any BioJulia member fulfilling
+this, may offer to take on this [responsibility](#biojulia-administrators).
+
+## Styleguides
+
+### Git Commit messages
+
+* Use the present tense ("Add feature" not "Added feature").
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...").
+* Limit the first line to 72 characters or less.
+* Reference issues and pull requests liberally after the first line.
+* Consider starting the commit message with an applicable emoji:
+    * :art: `:art:` when improving the format/structure of the code
+    * :racehorse: `:racehorse:` when improving performance
+    * :memo: `:memo:` when writing docs
+    * :penguin: `:penguin:` when fixing something on Linux
+    * :apple: `:apple:` when fixing something on macOS
+    * :checkered_flag: `:checkered_flag:` when fixing something on Windows
+    * :bug: `:bug:` when fixing a bug
+    * :fire: `:fire:` when removing code or files
+    * :green_heart: `:green_heart:` when fixing the CI build
+    * :white_check_mark: `:white_check_mark:` when adding tests
+    * :arrow_up: `:arrow_up:` when upgrading dependencies
+    * :arrow_down: `:arrow_down:` when downgrading dependencies
+    * :exclamation: `:exclamation:` when removing warnings or depreciations
+
+### Additional julia style suggestions
+
+* Source code files should have the following style of header:
+
+  ```julia
+  # Title
+  # =====
+  #
+  # Short description.
+  #
+  # [Long description (optional)]
+  #
+  # This file is a part of BioJulia. License is MIT: <link to the license file>
+  ```
+
+* Indent with 4 spaces.
+
+* When returning values from a function, use an explicit `return` statement.
```

Comment:
Should this be part of the guidelines?  It seems like it might be controversial, and for short-format function definitions, `return` is almost never used.